crates/workspace/src/item.rs

```diff
@@ -75,6 +76,10 @@ pub struct ItemSettingsContent {
     ///
     /// Default: right
     close_position: Option<ClosePosition>,
+    /// Whether to show file icons in tabs.
+    ///
+    /// Default: true
+    file_icons: Option<bool>,
```

Comment:
This is now always default false right? So we are missing here a default value override.

Reply:
isn't it default true? or wdym?

Reply:
Nope the default value is false.

https://doc.rust-lang.org/std/default/trait.Default.html#tymethod.default

Reply:
hm I see, but it doesn't look like other fields do this override when their default is also true, e.g line 91?

Reply:
Yea, I don't think we need to do anything here, as the default value will come from `default.json`.